### DISCUSSION TITLE
How find LossLess-Cut version?

### DISCUSSION THREAD

**Discussion by dacorsa** [mifi/lossless-cut#531] [Q&A]
Hi
i'm trying to see the installed version of LossLess-Cut from COMMAND LINE:

./losslesscut -v
./losslesscut -V
./losslesscut --version

There is no README, CHANGELOG in the package?
Thanks, can you helpme?

**Comment by mifi** [comment]
Launch it, then use the help menu

**Comment by perdakovich** [comment]
Help -> About

![image](https://github.com/mifi/lossless-cut/assets/89806596/3b571941-7849-4181-bf9e-dd67ab8009a8)


Nothing there.

**Reply by mifi** [reply]
See https://github.com/mifi/lossless-cut/issues/1537

**Reply by mifi** [reply]
If u launch it from the command line it should print out version

**Reply by perdakovich** [reply]
https://github.com/mifi/lossless-cut/assets/89806596/e5771086-980c-42a8-bd05-fad1a1a1b33c

**Reply by mifi** [reply]
did you check the command line? i don't see it in the videoyou shared

**Reply by perdakovich** [reply]
![image](https://github.com/mifi/lossless-cut/assets/89806596/9fe53f84-cdac-440b-919c-16fd7621cc09)

Nothing here

**Reply by mifi** [reply]
ahh it seems to be only logged in development. i will enable it for prod too
